packages/web/src/components/FatalErrorBoundary.tsx

```diff
@@ -14,7 +25,15 @@ class InfallibleErrorBoundary extends React.Component {
   }
 }
 
-class FatalErrorBoundary extends React.Component {
+type PropsFatalErrorBoundary = {
+  children?: React.ReactNode
+  page: React.ComponentType
```

Comment:
I actually wanted to ask Peter about this, since pages haven't been typed yet (I believe the tracking issue is https://github.com/redwoodjs/redwood/issues/523). We may want to loop back here and update this once pages are typed, but in the meantime `React.ComponentType` is fine imo 👍